### DISCUSSION TITLE
Duplicates another audio project name

### DISCUSSION THREAD

**Discussion by lepicsaintloup** [Losses/rune#40] [General]
This is just a FYI: 
https://www.runeaudio.com

**Comment by Losses** [comment]
Thank you for the information. Naming projects is definitely not my strong suit, and having a player with the same pronunciation as Roon has caused some issues. Here’s my perspective on this matter:

Just like there are many people named Alice and Kevin in the world, as long as trademark registration is not involved, having the same name for open-source projects doesn’t seem to be a significant problem. 

When I saw this issue, many alternative names flashed through my mind, but they all coincidentally matched existing products. Considering the current heavy development workload, I’m really not keen on spending too much energy on naming.

However, I’m open to a new name. Before the project releases version 1.0, we have an opportunity to rename it. Community members are welcome to share your ideas.

**Comment by Losses** [comment]
At present, this seems more like a valuable discussion rather than a trouble or "issue." So let's turn this into a discussion. Thank you for your participation!
